.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Type of issue
+<!--
+Is this a *bug* report, a *feature* request, or a *question* or request for *support*?
+-->
+
+# Description / Steps to reproduce
+
+<!--
+- For features: A description of the feature.
+- For bugs: The steps to reproduce the issue.
+-->
+
+# Steps to reproduce / Link to reproduction test-case
+
+<!--
+Please link to a test-case for easier reproduction of the issue, for example a RunKit playground or a repository with a `packages.json` that installs all required dependencies and is able to execute the script.
+
+Note: Failure to provide a test-case for reproduction purposes will result in the issue being closed.
```

Comment:
This one is too strict.

`If possible, please create a reproduction...` or smth like this is probably a better and a more realistic one.

Reply:
I'm maintaining an open source project as well. Getting bug reports without a means to reproduce the bug can be really quite frustrating, and often leads to dangling issues that never get resolved. Most of them end up being closed unresolved... Making it sound optional will maybe not be enough of an encouragement? People can still erase these templates and write whatever they want anyway, and many of them do. I'll leave it to @koskimas to decide how determined the wording should be.

Reply:
@VladShcherbin If some kind of reproduction is not required, I end up writing them myself. Something like 95% of the bug reports end up being user errors. That is all out of my time to fix actual bugs and write new features. Do you realize that I do this on my free time for free? We need more contributors or we need to enforce some kind of reproduction. If an issue is clearly a bug, It won't get closed, but issues like "eager doesn't work!!!11! Fix iT!!11!" will be closed.

Reply:
@koskimas I meant the last part, after `Note`. As a developer, I'd be very sad to read that if I don't add a reproduction repo than issue will be closed. This sounds very unfriendly to me.

Reply:
@VladShcherbin As a maintainer it is sad to leave open hundreds of issues that cannot be reproduced and might be real errors or not. If the issue seems critical, I'm sure that contributors investigate it anyways. Also if they are real issues they will raise from noise of closed issues, since multiple people will report the same thing over and over again.

Reply:
@VladShcherbin Yeah I think the wording could be less strict there. That seems harsh, even though it is true 😄

Reply:
Something like "Issues with a reproduction get solved much faster", or "Issues without a reproduction can get ignored for a long time until someone else finds time to reproduce them"

Reply:
👍